src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -147,6 +147,11 @@ public void setPerson(Person target, Person editedPerson) {
         public void updateFilteredPersonList(Predicate<Person> predicate) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public void viewDriverTask(Person target) {
+            throw new AssertionError("This method should not be called.");
+        }
```

Comment:
since this is for view command, then it should have its own view command test. i suggest creating the test after the view command is done

Reply:
my bad, this is for implementing methods for modelstub which extends model.